### DISCUSSION TITLE
How to connect kafka-ui docker with an installed kafka on mac running on plaintext

### DISCUSSION THREAD

**Discussion by rajjaiswalsaumya** [provectus/kafka-ui#4201] [Q&A]
I am trying to setup Kafka ui for the first time. From the docs, I was not able to figure out how can I make Kafka ui connect to my local broker running as installed binaries. Currently, Kafka ui is showing loading and then timing out. PS: I don't have kafka running on the default port and started kafka ui with dynamic config where I gave the bootstrap server URL with the correct port.

**Comment by Haarolean** [comment]
If you're running kafka on bare metal, take a look at [this](https://github.com/provectus/kafka-ui/discussions/1081). You'd need to use the internal docker host for this purpose, don't forget to set up the advertised listeners properly.
```
$  cat /etc/hosts | grep internal
127.0.0.1 kubernetes.docker.internal
```
